conf/chi.conf.example

```diff
@@ -17,7 +17,7 @@ expires_in=1d
 expires_on_backend=1
 
 [storage httpd.portal]
-expires_in=10m
+expires_in=1d
```

Comment:
1 hour seems small to me.

How 6 hours sound ?

After 6 hours it can be safe to assume the user will restart the whole process

After 1 hour the user may resume his browsing and try to complete the transaction.

We have to keep in mind that completing a transaction that we don't have a session for means the user threw money away and will require a refund by the PacketFence admin